.dockerignore

```diff
@@ -0,0 +1,6 @@
+/**
+!/generator.sh
```

Comment:
These lines that start with `!` list files and directories that will be copied into the docker image.